python/plugins/db_manager/dlg_table_properties.py

```diff
@@ -334,36 +337,27 @@ def deleteIndex(self):
 
     def createComment(self):
         """Adds a comment to the selected table"""
-
         try:
-            #Using the db connector, executing de SQL query Comment on table
-            self.db.connector._execute(None, 'COMMENT ON TABLE "{0}"."{1}" IS E\'{2}\';'.format(self.table.schema().name, self.table.name, self.viewComment.text()))
+            schem = self.table.schema().name
```

Comment:
Should not you check here if comments are supported?

Reply:
Well in fact the method run only if you click the button "add comment" is pressed.
The button is up only if it supports comment. 
So for me i don't need to add an other check in the method, but if you think it's better i can.